linear_programming.cpp

```diff
@@ -42,13 +41,12 @@ When flag=2: Unbounded
             min_bound = max(min_bound, bound);
     }
 
-    if ((*flag) == 1)
-        return -1;
     if (max_bound < min_bound)
```

Comment:
> > if (max_bound < min_bound)
> 
> I think it is infeasible solution and should be threated the same way as `a[i] == 0.0 && b[i] < 0`
> 
> In general it should never happen in the algorithm. If it happens that means something is wrong

Yes that is a bug. In MbC algorithm there should not be infeasible case. So it may not happen in MbC